pkgs/development/interpreters/perl/default.nix

```diff
@@ -118,6 +120,7 @@ let
           --replace "${
               if stdenv.cc.cc or null != null then stdenv.cc.cc else "/no-such-path"
             }" /no-such-path \
+          --replace "${stdenv.cc}" /no-such-path \
```

Comment:
This looks  very similar to the substitution above, which seems to do the same job for `x86_64-linux`.
What exactly is the difference between `stdenv.cc` and `stdenv.cc.cc` ?